File: java/eyes.playwright.java/src/main/java/com/applitools/eyes/playwright/universal/driver/dto/DriverInfoFeaturesDto.java
package com.applitools.eyes.playwright.universal.driver.dto;

import com.fasterxml.jackson.annotation.JsonInclude;

import java.util.Objects;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class DriverInfoFeaturesDto {
    private Boolean allCookies;
    private Boolean canExecuteOnlyFunctionScripts;

    public DriverInfoFeaturesDto() {
        this.allCookies = true;
        this.canExecuteOnlyFunctionScripts = true;
    }

    public Boolean getAllCookies() {
        return allCookies;
    }

    public void setAllCookies(Boolean allCookies) {
        this.allCookies = allCookies;
    }

    public Boolean getCanExecuteOnlyFunctionScripts() {
        return canExecuteOnlyFunctionScripts;
    }

    public void setCanExecuteOnlyFunctionScripts(Boolean canExecuteOnlyFunctionScripts) {
        this.canExecuteOnlyFunctionScripts = canExecuteOnlyFunctionScripts;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        DriverInfoFeaturesDto that = (DriverInfoFeaturesDto) o;
        return Objects.equals(allCookies, that.allCookies) && Objects.equals(canExecuteOnlyFunctionScripts, that.canExecuteOnlyFunctionScripts);
    }

    @Override
    public int hashCode() {
        return Objects.hash(allCookies, canExecuteOnlyFunctionScripts);
    }

    @Override
    public String toString() {
        return "DriverInfoFeaturesDto{" +
                "allCookies=" + allCookies +
                ", canExecuteOnlyFunctionScripts=" + canExecuteOnlyFunctionScripts +
                '}';
    }
}

File: dotnet/Eyes.Playwright.DotNet/Universal/Driver/Dto/DriverInfoFeaturesDto.cs
using Newtonsoft.Json;

namespace Applitools.Playwright.Universal.Driver.Dto
{
    public class DriverInfoFeaturesDto
    {
        [JsonProperty("allCookies")]
        public bool AllCookies { get; set; }
        
        [JsonProperty("canExecuteOnlyFunctionScripts")]
        public bool CanExecuteOnlyFunctionScripts { get; set; }

        public DriverInfoFeaturesDto()
        {
            AllCookies = true;
            CanExecuteOnlyFunctionScripts = true;
        }

        public override bool Equals(object o)
        {
            if (this == o)
            {
                return true;
            }

            if (!(o is DriverInfoFeaturesDto other))
            {
                return false;
            }

            return other.AllCookies.Equals(AllCookies) &&
                   other.CanExecuteOnlyFunctionScripts.Equals(CanExecuteOnlyFunctionScripts);
        }

        public override int GetHashCode()
        {
            return (AllCookies ? 1 : 0) + (CanExecuteOnlyFunctionScripts ? 2 : 0);
        }

        public override string ToString()
        {
            return "DriverInfoFeaturesDto{" +
                   "allCookies=" + AllCookies +
                   ", canExecuteOnlyFunctionScripts=" + CanExecuteOnlyFunctionScripts +
                   '}';
        }
    }
}